go.mod

```diff
@@ -1,9 +1,11 @@
 module github.com/goto/salt
 
-go 1.16
+go 1.21
```

Comment:
same reason as above

Reply:
I think that is because you are using go version 1.21 in your local.


it is okay, can you please check  in this repo, where we need to bump up the go version again?

Reply:
Right. It is okay. Lets use 1.21